Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/MainFragment.kt

```diff
@@ -74,6 +77,17 @@ class MainFragment : BaseFragment() {
         tracingViewModel.refreshActiveTracingDaysInRetentionPeriod()
         TimerHelper.checkManualKeyRetrievalTimer()
         submissionViewModel.refreshDeviceUIState()
+        main_scrollview.sendAccessibilityEvent(AccessibilityEvent.TYPE_VIEW_FOCUSED)
```

Comment:
please access all views via bindings
in this case it would be `binding.mainScrollView.xxx`

Reply:
that's relevant for all other bindings and files. I won't mark them